site/source/user-manual/service-guides/specter/specter-service.rst

```diff
@@ -1,14 +1,17 @@
 .. _specter-service:
 
-Specter Service
----------------
+====================
+Specter as a Service
```

Comment:
```suggestion
Specter Server
```

Let's avoid "as a service" - it has connotations

Reply:
I agree, the tradeoff here though is that I have to be very clear about when I'm referring to Specter running on the server vs Specter Desktop. It's a confusing setup - I'm going to research if the HWI can run as a daemon on the client device without using Specter Desktop now.